packages/fyndiq-icon-brand/styles.less

```diff
@@ -65,3 +71,10 @@
     fill: transparent;
   }
 }
+
+.tagline {
+  fill: @red;
+  text-anchor: middle;
+  font-size: 6.5px;
```

Comment:
I took for reference the mobile website haha.

Since the 2 websites are already not consistent, I can add a prop to the component, like `taglineSize` which would be the size in `px`. And default would be either 6.5px or 5.5